.devcontainer/devcontainer.json

```diff
@@ -3,24 +3,20 @@
     "name": "ros2_rust_roscon_spain:latest",
     "build": {
     "context": "..",
-    "dockerfile": "../.docker/Dockerfile",
+    "dockerfile": "../docker/Dockerfile",
     "args": {
       "XDG_RUNTIME_DIR": "${localEnv:XDG_RUNTIME_DIR}"
     }
     },
-    "workspaceFolder": "/ros2_rust_ws",
     "runArgs": [
     "--network=host",
-    "--hostname=devcontainer",
     "--privileged"
     ],
     "containerEnv": {
       "DISPLAY": "${localEnv:DISPLAY}",
       "TERM": "xterm-256color"
     },
-    "mounts": [
-      "source=${localWorkspaceFolder},target=/ros2_rust_ws/src/rust_ws_docs,type=bind"
-    ],
```

Comment:
All this is unnecessary if we just go with the default

Reply:
I agree! Thank you!